analyzers/src/SonarAnalyzer.VisualBasic/Rules/BypassingAccessibility.cs

```diff
@@ -31,21 +30,7 @@ namespace SonarAnalyzer.Rules.VisualBasic
     [Rule(DiagnosticId)]
     public sealed class BypassingAccessibility : BypassingAccessibilityBase<SyntaxKind>
     {
-        private static readonly DiagnosticDescriptor rule =
-            DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager)
-                .WithNotConfigurable();
-
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } =
-            ImmutableArray.Create(rule);
-
-        public BypassingAccessibility()
-            : this(AnalyzerConfiguration.Hotspot)
-        {
-        }
```

Comment:
I think this change makes sense since this rule was previously a hotspot and it was changed to code smell. I also think this deserves a separate PR.